packages/chat-client/package.json

```
@@ -29,17 +29,24 @@
     "format:scripts": "prettier . --write"
   },
   "dependencies": {
+    "@ethersproject/address": "^5.7.0",
+    "@noble/ed25519": "^1.7.1",
     "@walletconnect/core": "^2.1.4",
```

Latest is 2.3.3 so we might as well bump core here too:

```suggestion
    "@walletconnect/core": "^2.3.3",
```

---

also `@walletconnect/types` but it's out of range for comment/suggestion here annoyingly